src/main/java/org/folio/services/fiscalyear/FiscalYearApiService.java

```diff
@@ -68,11 +68,7 @@ public Future<FiscalYear> getFiscalYearById(String id, boolean withFinancialSumm
   }
 
   public Future<Void> updateFiscalYear(FiscalYear fiscalYear, RequestContext requestContext) {
-    return configurationEntriesService.getSystemCurrency(requestContext)
-      .compose(currency -> {
-        fiscalYear.setCurrency(currency);
-        return fiscalYearService.updateFiscalYear(fiscalYear, requestContext);
-      });
+    return fiscalYearService.updateFiscalYear(fiscalYear, requestContext);
```

Comment:
I think some more smart logic should be implemented here. Firstly need to get currecny from FiscalYear and only if it is null - get system currency

Reply:
Updated it